Mathport/Syntax/Transform/Tactic.lean

```diff
@@ -15,6 +15,14 @@ def transformConsecutiveTactics : Syntax → Syntax → M Syntax
     `(tactic| suffices $ty:term by $seq:tacticSeq)
   | `(tactic| have $[$id:ident]? $[: $ty:term]?), `(tactic|· $seq:tacticSeq) =>
     `(tactic| have $[$id:ident]? $[: $ty:term]? := by $seq:tacticSeq)
+  | `(tactic| have $[$id:ident]? $[: $ty:term]?), `(tactic|exact $t) =>
+    `(tactic| have $[$id:ident]? $[: $ty:term]? := $t)
+  | `(tactic| let $id:ident $[: $ty:term]?), `(tactic|· $seq:tacticSeq) =>
+    `(tactic| let $id:ident $[: $ty:term]? := by $seq:tacticSeq)
+  | `(tactic| suffices : $ty:term), `(tactic|· $seq:tacticSeq) =>
+    `(tactic| suffices $ty:term by $seq:tacticSeq)
+  | `(tactic| obtain $[$pat]? $[: $ty]?), `(tactic|· $seq:tacticSeq) =>
+    `(tactic| obtain $[$pat]? $[: $ty]? := by $seq:tacticSeq)
```

Comment:
I have also seen `by · $seq` show up a couple times, I'm not sure why. I think there is a special case for this in synport right now, but it would make more sense to remove the special case and add it as a transformer instead.

Reply:
Ok, I've added the rule.  I couldn't find the special case in synport (or at least not without accidentally breaking something) but I guess it is related to the `TacticContext` flag.

Reply:
Yeah, I think the flag isn't needed for the simpler translation. I'll take a look at simplifying it later, feel free to merge when ready.